plugins/modules/azure_rm_servicebus.py

```diff
@@ -47,6 +47,42 @@
             - basic
             - premium
         default: standard
+    identity:
+        description:
+            - Identity for this resource.
+        type: dict
+        version_added: '2.7.0'
+        suboptions:
+            type:
+                description:
+                    - Type of the managed identity
+                choices:
+                    - SystemAssigned
+                    - SystemAssigned, UserAssigned
+                    - UserAssigned
+                    - 'None'
+                default: 'None'
+                type: str
+            user_assigned_identities:
+                description:
+                    - User Assigned Managed Identities and its options
+                required: false
+                type: dict
+                default: {}
+                suboptions:
+                    id:
+                        description:
+                            - List of the user assigned identities IDs associated to the WebApp
+                        required: false
+                        type: list
+                        elements: str
+                        default: []
+                    append:
+                        description:
+                            - If the list of identities has to be appended to current identities (true) or if it has to replace current identities (false)
+                        required: false
+                        type: bool
+                        default: True
```

Comment:
Can we set `append` defaults to `false. The default behavior of the identities should be able added and removed not only added.

Reply:
I can change it but I was following the same default from azure_virtual_machine

https://github.com/ansible-collections/azure/blob/187b0c22ec3641d2b63db969509f0480352099f1/plugins/modules/azure_rm_virtualmachine.py#L492

which as been the default since 2023-06-15
```
ef944d9f (cpiment            2023-06-15 10:03:47 +0200  492)                         default: True
```

Reply:
My concern is that the default value of `append` is `true`, which means we can only add identities but not remove any. Users may be confused or they may simply treat it as a bug.

Reply:
Since we support `overwrite` and `append`, defaults to `overwrite`  would be better for users.

Reply:
I've changed it to default False.

We should put a PR through to change all of them to default False to be consistent.